### ISSUE TITLE
Craft 4: GraphQL errors when trying to retrieve data

### ISSUE THREAD

**Issue by tremby** [verbb/formie#839]
**Description**

I'm starting to explore what sort of output Formie will give over GraphQL, and I'm hitting errors.

**Steps to reproduce**
1. Make a form. Mine is a kitchen sink one, with pretty much one of everything.
2. Make queries over GraphQL for details of the form. Examples:
   - `{ formieForms { fields { id } } }` -> `Getting unknown property: verbb\\formie\\elements\\Form::fields`
   - `{ entries { ... on pages_default_Entry { id form { id fields { id } } } } }` -> `Getting unknown property: verbb\\formie\\elements\\Form::fields`
   - `{ entries { ... on pages_default_Entry { id form { id pages { id fields { id } } } } } }` -> `Getting unknown property: verbb\\formie\\models\\FieldLayoutPage::fields`
   - `{ entries { ... on pages_default_Entry { id form { id pages { id settings { id } } } } } }` -> `Getting unknown property: verbb\\formie\\models\\PageSettings::id`

### Form settings
Please pick from the options below.

- Multi-page form: Yes
- Submission Method: Ajax or Page Reload whatever is default
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: dev-craft-4 at df609e2
- Craft version: 4.0.x-dev at a59b90
- Multi-site: Yes

**Additional context**
Headless mode.

All of the queries I'm composing are built via the "explore" pane of Graphiql, i.e. these are all fields the schema claims to provide.

**Comment by engram-design** [comment]
Using `fields` is probably one of the few breaking changes for Craft 4 - this will be `formFields`. In Craft 3, we could get around using `fields`, but it's a protected name when it comes to GQL schema, so we're now running into issues with Craft 4. But we left it as-is due to it being a breaking change.

- `FormInterface::fields` is now `FormInterface::formFields`
- `PageInterface::fields` is now `PageInterface::pageFields`
- `RowInterface::fields` is now `RowInterface::rowFields`

As for `PageSettings::id` that's a valid error - there is no `id` property for `PageSettings`, but I have just noticed this is coming up in the explorer incorrectly.

Fixed in the `craft-4` branch.
